static/js/twitter.js

```diff
@@ -8,5 +8,43 @@
   js = d.createElement(s)
   js.id = id
   js.src = p + '://platform.twitter.com/widgets.js'
+  js.setAttribute('onload', 'twitter_filter()')
   fjs.parentNode.insertBefore(js, fjs)
 })(document, 'script', 'twitter-wjs')
+
+function twitter_filter() {
+  const node = obj => obj.parentNode.parentNode.parentNode.parentNode.parentNode
+  var twitter_handles = []
+  var orgs = document.getElementsByClassName('org-info')
+  for (var i = 0; i < orgs.length; i++) {
+    var org_info = orgs[i].children
+    var children = [].slice.call(org_info)
+    var twitter_url = children.filter(
+      x => x.nodeName === 'A' && x.host === 'twitter.com'
+    )
+    if (twitter_url.length != 0) {
```

Comment:
Please follow this guide <https://github.com/airbnb/javascript#comparison--eqeqeq> and give a blank line above `if` statement for readability.